homeassistant/components/sia/strings.json

```diff
@@ -0,0 +1,42 @@
+{
+  "title": "SIA Alarm Systems",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "port": "[%key:common::config_flow::data::port%]",
+          "account": "Account",
+          "encryption_key": "Encryption Key",
+          "ping_interval": "Ping Interval (min)",
+          "zones": "Number of zones for the account",
+          "ignore_timestamps": "Ignre the timestamp check of the SIA events",
```

Comment:
```suggestion
          "ignore_timestamps": "Ignore the timestamp check of the SIA events",
```

Reply:
This typo has not been corrected.

You fixed the same typo in line 32 https://github.com/home-assistant/core/pull/36625/commits/fd68a3d23120214e6555ccc4f4af5185c53d70cb#diff-a34d3090bb29f0c9b57188fd7488bb66b3f534aea7e4b459477f2badb19525c5R23